docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -73,19 +129,53 @@ the ``copy`` module::
     ...
     '''
 
-Building & Testing
+Formatting options
 ''''''''''''''''''
+These formatting functions are ``U()``, ``M()``, ``I()``, and ``C()``
+for URL, link to help for another module, `italic`, and ``constant-width`` respectively. It is suggested
+to use ``C()`` for file and option names, and ``I()`` when referencing
+parameters; module names should be specified as ``M(module)`` to create a
```

Comment:
jtyr has a PR open about this phrasing.  The use of "option names" and "parameters" is confusing.  jtyr thinks (I haven't read thoroughly enough to know) that they are referring to the module's option names and example values for the options.  For instance:
```
- externaldata:
    description: If I(externaldata) is set to C(True) then I(datasource) must be set to the url to read the data from.
```

His PR suggests using "option names" and "option values" here.